src/app/speechbar.service.spec.ts

```diff
@@ -201,6 +205,89 @@ describe('SpeechbarService', () => {
       service.appendButton(mockButton, '+less');
     })();
   });
+
+  it('should handle backspace actions', done => {
```

Comment:
These tests are not for the functionality added, just for missing branch coverage in speechbar service